src/scr_fetch.c

```diff
@@ -774,14 +798,17 @@ static int scr_fetch_summary(
     /* the key is the source rank, which we don't care about,
      * the info we need is in the element hash */
     scr_hash* elem_hash = scr_hash_elem_hash(elem);
-
+    scr_dbg(1, "scr_fetch_summary print at %d:%s", __LINE__, __FILE__);
+    scr_hash_log(elem_hash, 1, 0);
     /* get pointer to file hash */
     scr_hash* file_hash = scr_hash_get(elem_hash, SCR_SUMMARY_6_KEY_FILE);
     if (file_hash != NULL) {
       /* TODO: parse summary file format */
+      scr_dbg(1, "scr_fetch_summary at %d:%s", __LINE__, __FILE__);
       scr_hash_merge(file_list, elem_hash);
     } else {
-      rc = SCR_FAILURE;
```

Comment:
My understanding here is that if a given rank did not have any files in a given checkpoint set (which seems reasonable), setting this failure means that upon fetching that checkpoint set, SCR would return an error.  I commented this out, tested it, and it seemed to match my understanding.  I decided to uncomment it because it seemed like an improvement.  My apologies for not including a comment here explaining it